todo_list/models.py

```diff
@@ -1,3 +1,21 @@
 from django.db import models
 
-# Create your models here.
+
+class Page(models.Model):
+    def __str__(self):
+        return self.title
+
+    title = models.CharField(max_length=30)
```

Comment:
@quratulain25 Do you think 30 char length is sufficient? Also while determining char length use 2^x formula.

Reply:
@ifzal , what is x?